Package generator guide update. (#16418)
ckeditor/ckeditor5: docs/framework/development-tools/package-generator/using-package-generator.md

* `--plugin-name` &ndash; define the plugin name to be different from the package name.
* `--lang` &ndash; (values: `js` | `ts`) choose whether the created package should use JavaScript or TypeScript. If omitted, the script will ask the user to choose manually.
* `--verbose` &ndash; (alias: `-v`) print additional logs about the current executed task.
* `--use-only-new-installation-methods` &ndash; use only new installation methods in a newly created package.

## Choosing the method

Currently there are two modes of operation for package generator:

> * the default way is based on the {@link getting-started/legacy-getting-started/quick-start legacy installation methods} and can build packages that support both current installation approach, the legacy one with DLLs and both at once.

Reviewer: ```suggestion
* By default, package generator creates bundles for both the current installation methods and {@link getting-started/legacy-getting-started/quick-start legacy installation methods}.
```

Reviewer 2: How about we drop the link from here?

> * using the `--use-only-new-installation-methods` flag will base the package on imports from the `ckeditor5` package only. It provides no support for legacy installation methods.

Reviewer: ```suggestion
* When the project is generated with the `--use-only-new-installation-methods` flag, the package will only support the current installation methods.
```

> * When the project is generated with the `--use-only-new-installation-methods` flag, the package will only support the current installation methods. This approach makes it easier to create CKEditor 5 plugin at the cost of not supporting the old installation methods.

Reviewer 2: ```suggestion
* When the project is generated with the `--use-only-new-installation-methods` flag, the package will only support the current installation methods. This approach makes it easier to create CKEditor&nbsp;5 plugin at the cost of not supporting the old installation methods.
```
